framework/src/vectorpostprocessors/LineValueSampler.C

```diff
@@ -27,11 +29,16 @@ validParams<LineValueSampler>()
 }
 
 LineValueSampler::LineValueSampler(const InputParameters & parameters)
-  : PointSamplerBase(parameters)
+  : PointSamplerBase(parameters),
+    _line_vector(getParam<Point>("end_point") - getParam<Point>("start_point")),
+    _line_vector_norm(_line_vector.norm())
 {
   Point start_point = getParam<Point>("start_point");
   Point end_point = getParam<Point>("end_point");
 
+  if (_line_vector_norm == 0.0)
```

Comment:
I'd use a fuzzy comparison here:
```
if MooseUtils::absoluteFuzzyEqual(_line_vector_norm, 0.0)

Reply:
I have made this change. Thanks for catching this error.